tests/testflows/rbac/requirements/requirements.py

```diff
@@ -0,0 +1,6132 @@
+# These requirements were auto generated
```

Comment:
This file was generated from some file named requirements.md as far as I understood. Could you add that source file to our repository as well?

Reply:
Ok, I will ask @alex-zaitsev if we can release the source for our internal RBAC requirements.